packages/expo-notifications/README.md

```diff
@@ -1,5 +1,115 @@
 # expo-notifications
 
-`expo-notifications` provides some great stuff.
+## Instalation 
+
+`expo-notifications` needs `react-native-unimodule` to work. You can read more about it [here](https://www.npmjs.com/package/react-native-unimodules). 
+
+Add `expo-notifications` to your project:
+execute the following command in your project root directory:
+"yarn add expo-notifications".
+
+### Android
+
+#### 1. Modify AndroidManifest
+
+`AndroidManifest.xml` is a place where you can configure `expo-notifications` module. For Instance, you can choose there
+if you want to use Expo's servers as a middleman or choose that you don't want to use push notifications at all.
+
+Configuration template:
+
+```xml
+
+<!-- expo-notifications START -->
+    <!-- turn on push notifications START -->
```

Comment:
I'm not sure this will be helpful for the users, but maybe it's just my opinion. I would prefer to read a README where I could see:
- for every feature an explanation what to add to `AndroidManifest` (so eg. single `service` node, single `meta-data` node)
- an example section with placeholders filled in.

You kind of already did it when designing the _use cases_ section, I just think providing configuration example for every _use case_ would be readable. 🙂